.azure/pipelines/ci.yml

```diff
@@ -576,16 +576,18 @@ stages:
         displayName: Pack Templates
       - script: ./src/ProjectTemplates/build.cmd -ci -test -NoRestore -NoBuild -NoBuilddeps "/p:RunTemplateTests=true /bl:artifacts/log/template.test.binlog"
         displayName: Test Templates
+      - script: ./src/Grpc/build.cmd -ci -test "/bl:artifacts/log/grpc.test.binlog"
```

Comment:
If grpc tests work on helix natively, I'd recommend not adding them to the azdo test legs, and leaving this as soley for Windows_templates_tests, and you can potentially remove any of the non helix codepaths (if any) as well then.

Reply:
Actually I guess you still want the non helix code paths for local dotnet test runs

Reply:
There's still a need to build the shared framework before we run these interop tests which is why we put the tests here. The goal here is to future proof them so it's easy to run them on Helix as @Tratcher suggested.

Reply:
So these tests are already running on helix with this PR right?  I'm just asking why do we need to add these to the non helix test job if they are passing on helix.

Reply:
Fair, I'll remove this once the tests are brought up on Helix.